graphql-schemas/schema.yml

```diff
@@ -34,6 +34,12 @@ confs:
   - { name: jiralert, type: JiralertSettings_v1 }
   - { name: customMessages, type: AppInterfaceCustomMessage_v1, isList: true }
   - { name: costReport, type: CostReportSettings_v1 }
+  - { name: terraformResourcesProviderExclusionsByProvisioner, type: TerraformResourcesProviderExclusionsByProvisioner_v1, isList: true}
+
+- name: TerraformResourcesProviderExclusionsByProvisioner_v1
+  fields:
+  - { name: provisioner, type: ExternalResourcesProvisioner_v1, isInterface: true, isRequired: True}
+  - { name: excludedProviders, type: string, isList: true, isRequired: true}
```

Comment:
`provisioner` and `excludedProviders` are not required in json schema file.

Reply:
fixed.